ccd-definition/CaseEventToFields/CareSupervision/uploadDocuments.json

```diff
@@ -9,7 +9,8 @@
     "PageID": 1,
     "PageDisplayOrder": 1,
     "PageColumnNumber": 1,
-    "CallBackURLMidEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/upload-documents/mid-event"
+    "CallBackURLMidEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/upload-documents/mid-event",
+    "ShowSummaryChangeOption": "Y"
```

Comment:
Don't think we want this label shown in the CYA

Reply:
ohh, yes agreed have removed the borders/labels etc